models/instrument.py

```diff
@@ -1202,6 +1202,16 @@ def get_short_filter_name(cls, filter):
 
         return filter
 
+    @classmethod
+    def get_full_filter_name(cls, shortfilter):
```

Comment:
In Exposure there's a method `filter_full` that returns the long filter name.  Here something "full" returns the short filter name.  We should have consistent naming to avoid confusion.  Perhaps this function should be `get_canonical_filter_name` or something like that?  I would stick with using "full" to refer only to long filter names.

Reply:
We discussed this in slack - the name is good, the behavior in `instrument.py` is confusing because by default it spits back out the short filter until it is overridden by an instrument. I'll add a comment to make this less confusing.